Dockerfile

```diff
@@ -206,6 +208,14 @@ RUN git clone https://github.com/go-swagger/go-swagger.git /go/src/github.com/go
 	&& (cd /go/src/github.com/go-swagger/go-swagger && git checkout -q $GO_SWAGGER_COMMIT) \
 	&& go install -v github.com/go-swagger/go-swagger/cmd/swagger
 
+# Install shellcheck from source to get a more recent version on debian jessie
+ENV SHELLCHECK_VERSION v0.4.6
+RUN set -x \
+	&& git clone --branch "$SHELLCHECK_VERSION" https://github.com/koalaman/shellcheck.git /usr/local/shellcheck \
+	&& cd /usr/local/shellcheck \
+	&& export LANG="C.UTF-8" && cabal update && cabal install
```

Comment:
Maybe changing `LANG` here globally might cause some trouble in later steps.
Does 
```bash
&& LANG="C.UTF-8" cabal update
&& LANG="C.UTF-8" cabal install
```
do the job as well?